src/test/java/school/hei/haapi/integration/TeacherIT.java

```diff
@@ -136,6 +142,34 @@ public static Teacher someCreatableTeacher() {
     return teacher;
   }
 
+  static CrupdateCourse crupdatedCourse1() {
```

Comment:
Change into inline variable. 

return new CrupdateCourse()
.id(COURSE5_ID)
.code("MGT1")
... 
etc


And do this for every object